giskard_vision/image_classification/dataloaders/loaders.py

```diff
@@ -76,3 +77,85 @@ def get_labels(self, idx: int) -> Optional[np.ndarray]:
         row = self.get_row(idx)
 
         return np.array([row[self.label_key]])
+
+
+class DataLoaderSkinCancerHuggingFaceDataset(DataLoaderHuggingFaceDataset):
+    """
+    A data loader for the `marmal88/skin_cancer` dataset on HF, extending the DataLoaderHuggingFaceDatasets class.
+
+    Attributes:
+        label_key (str): Key for labels in the dataset.
+        image_key (str): Key for accessing images in the dataset.
+
+    Args:
+        name (Optional[str]): Name of the data loader instance.
+        dataset_config (str): Specifies the dataset config, defaulting to "train".
+        dataset_split (str): Specifies the dataset split, defaulting to "train".
+    """
+
+    label_key = "dx"
+    image_key = "image"
```

Comment:
do we need it as attr if it's only used in `get_image`?

Reply:
if it's something standard across all HF datasets, let's move it to base, otherwise, let's remove it from the attrs

Reply:
Done